.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
```

Comment:
Either capitalise all the important words in the header or only capitalise the first one; either way, remain consistent throughout (this applies to headers below as well)